hack/tagger/go.mod

```diff
@@ -4,18 +4,4 @@ go 1.16
 
 require github.com/google/go-containerregistry v0.6.0
 
-require (
-	github.com/containerd/stargz-snapshotter/estargz v0.7.0 // indirect
-	github.com/docker/cli v20.10.7+incompatible // indirect
-	github.com/docker/distribution v2.7.1+incompatible // indirect
-	github.com/docker/docker v20.10.7+incompatible // indirect
-	github.com/docker/docker-credential-helpers v0.6.3 // indirect
-	github.com/golang/snappy v0.0.3 // indirect
-	github.com/klauspost/compress v1.13.0 // indirect
-	github.com/opencontainers/go-digest v1.0.0 // indirect
-	github.com/opencontainers/image-spec v1.0.1 // indirect
-	github.com/pkg/errors v0.9.1 // indirect
-	github.com/sirupsen/logrus v1.8.1 // indirect
-	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c // indirect
-	golang.org/x/sys v0.0.0-20210603125802-9665404d3644 // indirect
-)
+require github.com/sirupsen/logrus v1.8.1
```

Comment:
I'm wondering if this should go as part of this PR. I'm okay with the change, looks like this is valid, I tried to do `go mod tidy` inside `hack/tagger/` directory and it left the above file unchanged, so looks like we just need one direct dependency and not many indirect ones

Reply:
After running `make lint` to check in my local these changes occurred.

Reply:
Yes, `make lint` does `make get-deps` first